src/cmds/supportGuildSendAnn.cjs

```diff
@@ -0,0 +1,31 @@
+// This is a private command that can send the message to the channel.
+const { SlashCommandBuilder } = require('@discordjs/builders');
+
+const { bot, supportGuild } = require('../constants.json');
+
+const config = require('../../config');
+const ann = require('../data/latestUpdate');
+
+const updateAnnData = new SlashCommandBuilder()
+  .setName('sendann')
```

Comment:
well, uh this isnt a really great name "sendann". couldnt "annoucement" or such would be a better naming?